sympy/algebras/quaternion.py

```diff
@@ -10,10 +10,50 @@
 from sympy.core.sympify import sympify, _sympify
 from sympy.core.expr import Expr
 from sympy.core.logic import fuzzy_not, fuzzy_or
+from sympy.core.numbers import pi
 
 from mpmath.libmp.libmpf import prec_to_dps
 
 
+def _elementary_axis(axis):
+    if axis == 'x':
+        return [1, 0, 0]
+    if axis == 'y':
+        return [0, 1, 0]
+    if axis == 'z':
+        return [0, 0, 1]
+
+def _elementary_axis_index(axis):
+    if axis == 'x': return 1
+    if axis == 'y': return 2
+    if axis == 'z': return 3
```

Comment:
```suggestion
    return 'xyz'.index(axis) + 1
```

Reply:
Same as for _elementary_axis! Since they are oneliners, I wonder if it isn't better to just put them on their calls instead of defining these functions.

I think I'm gonna do that instead, to lower the diff.